assets/js/scripts.js

```diff
@@ -11,21 +17,28 @@
     return array;
   }
 
-  function setup(data) {
-    const shuffledData = shuffleArray(data);
+  async function setup() {
+    let fontData = await getJSONData();
+    let shuffledData = shuffleArray(fontData);
     const parent = document.getElementById("projectGrid");
 
     for (let i = 0; i < shuffledData.length; i++) {
+      let path = window.location.pathname;
       const project = shuffledData[i];
+
       const singleProject = document.createElement("a");
-      singleProject.href = project.url;
+      if (path == '/index.html') {
+        singleProject.href = project.url;
+      } else {
+        singleProject.href = `../${project.url}`;
+      }
       singleProject.setAttribute("id", i);
 
       const img = document.createElement("img");
-      if (project.img !== false) {
+      if (path == '/index.html') {
         img.src = `./assets/img/${project.img}`;
       } else {
-        img.src = "./assets/img/placeholder.jpg";
+        img.src = `../../assets/img/${project.img}`;
```

Comment:
The paths for the `img.src` and `singleProject.href` (on lines 30-34) are different depending on whether the user is on the index page or a project page.